presto-product-tests/src/main/java/io/prestosql/tests/TestGroups.java

```diff
@@ -33,17 +32,7 @@
     public static final String QUERY_ENGINE = "qe";
     public static final String COMPARISON = "comparison";
     public static final String LOGICAL = "logical";
-    public static final String SET_OPERATION = "set_operation";
     public static final String JSON_FUNCTIONS = "json_functions";
-    public static final String URL_FUNCTIONS = "url_functions";
-    public static final String ARRAY_FUNCTIONS = "array_functions";
-    public static final String BINARY_FUNCTIONS = "binary_functions";
-    public static final String CONVERSION_FUNCTIONS = "conversion_functions";
-    public static final String HOROLOGY_FUNCTIONS = "horology_functions";
-    public static final String MAP_FUNCTIONS = "map_functions";
-    public static final String REGEX_FUNCTIONS = "regex_functions";
-    public static final String STRING_FUNCTIONS = "string_functions";
```

Comment:
this groups were used in convention tests, maybe you want to remove them from there as well.

Reply:
Indeed. It's a mess. Some test convention tests' groups are not defined here (eg postgresql, an important example, as it's used to select tests for singlenode-postgresql env).

i remove these here to clear annoying compiler warning mostly. I don't think it's worthy keeping them, but if you see a value in this, I can restore them here.